includes/validators/class-users.php

```diff
@@ -7,52 +7,29 @@
 
 namespace Press_Sync\validators;
 
+use Press_Sync\validation\User as LocalUserData;
+use Press_Sync\api\route\validation\User as RemoteUserData;
 /**
  * User validation class to get and compare results.
  */
 class Users extends Validation_Utility implements Validation_Interface {
 	protected static $endpoint = 'users';
 
+	public function __construct() {
+		$this->local_data  = new LocalUserData();
+		$this->remote_data = new RemoteUserData();
```

Comment:
Hmmm...not sure we'd want to be instantiating a second instance of the API route object. Let's discuss.